### DISCUSSION TITLE
how do I apply a rego exception for terraform files?

### DISCUSSION THREAD

**Discussion by ryanpodonnell1** [aquasecurity/trivy#5206] [Q&A] [answered]
### Question

Trying to follow [this example](https://aquasecurity.github.io/trivy/v0.45/docs/scanner/misconfiguration/policy/exceptions/)  to apply a global exception for `AVD-AWS-0057` 

my attempt at a rego policy for this

```rego
package user.aws

exception[rules] {
        rules = ["AVD-AWS-0057"]
}

```

full command `trivy config . --config-policy ./rego`

### Target

Filesystem

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Operating System

MacOS

**Comment by nikpivkin** [comment, marked as answer]
Hi @ryanpodonnell1 !

The `ignore-policy` flag is not available for the `config` command, but you can scan the config file with the `fs` command and specify the `--scanners config` flag.  For example: ```trivy fs --scanners config . --ignore-policy path/to/foo.rego```

Note that the input structure is the same as the Trivy JSON output. https://aquasecurity.github.io/trivy/v0.45/docs/configuration/filtering/#by-open-policy-agent

You can also filter the result using [ignore files](https://aquasecurity.github.io/trivy/v0.45/docs/configuration/filtering/#by-finding-ids).

**Reply by simar7** [reply]
In addition to what @nikpivkin has suggested - we have an issue outstanding to track addition of `ignore-policy` to the `config` flag. https://github.com/aquasecurity/trivy/issues/3490
